libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -156,15 +165,32 @@ public override object VisitStructuredTemplateBody([NotNull] LGTemplateParser.St
             return result;
         }
 
+        /// <summary>
+        /// Visit a parse tree produced by the <c>normalBody</c>
+        /// labeled alternative in <see cref="LGTemplateParser.body"/>.
+        /// </summary>
+        /// <param name="context">The parse tree.</param>
+        /// <returns>An object of the visitor result.</returns>
```

Comment:
>An object of the visitor result. [](start = 21, length = 32)

This is really awkward phrasing.  I would just do "Visitor result." for all of these.  You don't need to add the type information.

Reply:
Got it. I changed all these occurracnes to "Visitor result".